test/Constraints/iuo_objc.swift

```diff
@@ -5,9 +5,11 @@ import Foundation
 
 func iuo_error(prop: IUOProperty) {
   let _: Coat? = prop.iuo.optional()
-  // expected-error@-1 {{value of optional type '(() -> Coat?)?' not unwrapped; did you mean to use '!' or '?'?}}
+  // expected-error@-1 {{value of optional type 'OptionalMethods?' not unwrapped; did you mean to use '!' or '?'?}}
```

Comment:
Is the promotion of the chain from `U!` to `T?` intended?

Reply:
Sorry is this comment pointed at the wrong place? Not sure what's intended.